lib/api/src/js/module.rs

```diff
@@ -194,7 +163,11 @@ impl Module {
     /// Opposed to [`Module::new`], this function is not compatible with
     /// the WebAssembly text format (if the "wat" feature is enabled for
     /// this crate).
-    pub fn from_binary(_store: &impl AsStoreRef, binary: &[u8]) -> Result<Self, CompileError> {
+    pub fn from_binary(
+        _store: &impl AsStoreRef,
+        binary: impl IntoBytes,
```

Comment:
We chatted about this before with @ptitSeb and @john-sharratt . Binary should be `&[u8]`